Shared/Supporting Files/Extensions/View+Sheet.swift

```diff
@@ -372,13 +372,13 @@ private struct Sheet<Content>: UIViewRepresentable where Content: View {
 }
 
 private extension Sheet {
-    class SheetModel<Content>: ObservableObject where Content: View {
+    class SheetModel<HostingContent>: ObservableObject where HostingContent: View {
```

Comment:
Since `HostingContent` is the same as `Content` could you just remove the generic parameter altogether?

Reply:
Good call, there is no need to have that parameter.